app/models/rook.rb

```diff
@@ -1 +1,13 @@
-class Rook < Piece; end
+class Rook < Piece
+  def valid_move?(x_destination, y_destination)
+    (vertical(x_destination, y_destination) || horizontal(x_destination, y_destination)) && !obstructed?(x_destination, y_destination)
+  end
+
+  def vertical(x_destination, y_destination)
```

Comment:
vertical and horizontal can be private methods

More info: http://culttt.com/2015/06/03/the-difference-between-public-protected-and-private-methods-in-ruby/